tests/test.php

```diff
@@ -1,18 +1,40 @@
 <?php
 require "../vendor/autoload.php";
 
+use app\models\ActionCancel;
+use app\models\ActionComplete;
+use app\models\ActionRefuse;
+use app\models\ActionRespond;
 use app\models\Task;
 
-$testTask = new Task(503, 354);
-echo "Проверка: getNewStatus(Task::ACTION_COMPLETE) <br>Ожидаем: completed<br>Результат: ";
-echo $testTask->getNewStatus(Task::ACTION_COMPLETE);
+$testTask = new Task(503, 533, 503); // id юзера, id заказчика, id исполнителя
+echo "Проверка: getNewStatus(Task::ACTION_COMPLETE) - ";
+echo $testTask->getNewStatus(Task::ACTION_COMPLETE) === Task::STATUS_COMPLETED;
 echo "<br>";
-echo "Проверка: getNewStatus(Task::ACTION_CANCEL) <br>Ожидаем: cancelled<br>Результат: ";
-echo $testTask->getNewStatus(Task::ACTION_CANCEL);
+echo "Проверка: getNewStatus(Task::ACTION_CANCEL) - ";
+echo $testTask->getNewStatus(Task::ACTION_CANCEL) === Task::STATUS_CANCELLED;
 echo "<br><br>";
 
-echo "Проверка: getAvailableActions(Task::STATUS_IN_PROCESS) <br>Ожидаем: cancel, respond <br> Результат: ";
-var_dump($testTask->getAvailableActions(Task::STATUS_NEW));
+echo "Проверка: getAvailableActions(Task::STATUS_NEW), пользователь: заказчик - ";
+print_r($testTask->getAvailableActions(Task::STATUS_NEW, $testTask->userId, $testTask->customerId, $testTask->executorId) == new ActionCancel);
```

Comment:
для проверки что объект является экземпляром конкретного класса надо использовать оператор `instanceof`:
```suggestion
print_r($testTask->getAvailableActions(Task::STATUS_NEW, $testTask->userId, $testTask->customerId, $testTask->executorId) instanceof ActionCancel);
```